composer.json

```diff
@@ -50,7 +56,7 @@
         "yiisoft/cache": "^3.0@dev",
```

Comment:
Do we use cache anywhere in the package?

Reply:
done.